spec/models/benefitting_country_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+RSpec.describe BenefittingCountry do
+  describe ".all" do
+    subject { described_class.all }
+
+    it "returns all the countries" do
+      expect(subject.count).to eq(177)
+    end
+
+    it "allows grouping by most specific region" do
```

Comment:
I would like this spec to better document what is happening here if we can. I do wonder if forcing the order would do that?